README.md

```diff
@@ -10,6 +10,13 @@ With MariaDB:
     % brew tap mroonga/mroonga
     % brew install mroonga --use-homebrew-mariadb
 
+If you have an old mysql, please switch to the latest version:
+
+    % mysql.server stop
+    % brew switch mysql 5.7.16
+    % brew link --overwrite mysql
```

Comment:
`brew link --overwrite <Some Formula name>` is dangerous operation.
Is there any safe way to achieve this?